modules/apps/product-navigation/product-navigation-applications-menu-web/src/main/resources/META-INF/resources/js/ApplicationsMenu.js

```diff
@@ -22,10 +22,15 @@ import '../css/ApplicationsMenu.scss';
 import ClayLabel from '@clayui/label';
 import ClaySticker from '@clayui/sticker';
 import ClayTabs from '@clayui/tabs';
+import {useEventListener} from 'frontend-js-react-web';
 import {fetch, navigate, openSelectionModal} from 'frontend-js-web';
 import PropTypes from 'prop-types';
 import React, {useRef, useState} from 'react';
 
+const OPEN_MENU_TITLE_TPL = `<div>${Liferay.Language.get(
+	'open-menu'
+)}</div><kbd class="c-kbd c-kbd-dark"><kbd class="c-kbd">⌘</kbd><span class="c-kbd-separator">+</span><kbd class="c-kbd">⇧</kbd><span class="c-kbd-separator">+</span><kbd class="c-kbd">M</kbd></kbd>`;
+
```

Comment:
Since we are having problems with spaces, maybe using the single quote to try to break lines could help with legibility a little. This is really not the best option I prefer the template literal, but I think it is better than having an inline markup here.

```js
const OPEN_MENU_TITLE_TPL = `<div>${Liferay.Language.get('open-menu')}</div>`
	+ '<kbd class="c-kbd c-kbd-dark">'
		+ '<kbd class="c-kbd">⌘</kbd>'
		+ '<span class="c-kbd-separator">+</span>'
		+ '<kbd class="c-kbd">⇧</kbd>'
		+ '<span class="c-kbd-separator">+</span>'	
		+ '<kbd class="c-kbd">M</kbd>'
	+ '</kbd>';
```

Reply:
I like it this way as well. Is our `yarn format` gonna like it too?

Reply:
Another option to keep it pretty would just create and array of stings to get the format and then `join()` together

Reply:
Matu's suggestion looks fine to the formatter and It's running well. I'll use it.